Documentation/InputSystem/HandTracking.md

```diff
@@ -161,3 +161,41 @@ public class MyHandMeshEventHandler : IMixedRealityHandMeshHandler
   }
 }
 ```
+
+## Known Issues
+
+### .NET Native
+
+There is currently a known issue with Master builds using the .NET backend. In .NET Native, `IInspectable` pointers cannot be marshaled from native to managed code using `Marshal.GetObjectForIUnknown`. The MRTK uses this to obtain the `SpatialCoordinateSystem` in order to receive hand and eye data from the platform.
+
+We've provided DLL source as a workaround for this issue, in [the native Mixed Reality Toolkit repo](https://github.com/microsoft/MixedRealityToolkit/tree/master/DotNetNativeWorkaround). Please follow the instructions in the README there and copy the resulting binaries into a Plugins folder in your Unity assets. After that, the WindowsMixedRealityUtilities script provided in the MRTK will resolve the workaround for you.
```

Comment:
We've provided a DLL and C# wrapper/utility/source as a workaround for this issue

Does WindowsMixedRealityUtilities show up as xref?

Reply:
To be honest I have no idea where this came from haha. @keveleigh is this because this PR https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5458/files is not in stabilization?